src/test/e2e/swagger/utils/helpers.ts

```
@@ -0,0 +1,68 @@
+  message: string;
+  code: string;
+  details?: {
+    [key: string]: string;
+  };
+}
+
+export async function interceptWithErrorResponse(
+  page: Page,
+  url: string,
+  errorResponse: ErrorResponse,
+  status: number = 400
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)